pkg/monitortests/node/legacynodemonitortests/exclusions.go

```diff
@@ -0,0 +1,26 @@
+package legacynodemonitortests
+
+import "regexp"
+
+func isThisContainerRestartExcluded(locator string) bool {
```

Comment:
we have different dimensions to create the list of exclusions.

1. topology: single node, HA, hypershift
2. architecture: amd64, arm64, multiarch
3. lowest installed version: don't enforce on levels that have installed 4.17 or below

Reply:
I'm not sure we expose that information at the event level.

Reply:
https://github.com/openshift/origin/blob/5c601963b6e077c89d4d44c2a2e4bed38f2ddd77/pkg/monitortestlibrary/platformidentification/types.go#L59

might be a place to get thsi info.